README.md

```diff
@@ -1,28 +1,217 @@
-    * either:
-      1. append a user defined odoo.conf to the one in the image (add this user defined
-      odoo.conf by Dockerfile or by volume)
-      2. or use ENV variables to customize parameters (I'm more in favor of that solution)
\ No newline at end of file
+# elicocorp/odoo
+Simple yet powerful Odoo image for Docker based on [OCB][ocb] code and
+maintained by [Elico Corp][ec].
+
+  [ocb]: https://github.com/OCA/OCB "Odoo Community Backports"
+  [ec]: https://www.elico-corp.com/
+
+This image is a fork of [XCG Consulting][xcg] Odoo Docker image available
+[here][xcgd].
+
+  [xcg]: https://www.xcg-consulting.fr/
+  [xcgd]: https://hub.docker.com/r/xcgd/odoo/
+
+## Usage
+In order to use this image, a recent version of [Docker][dk] must be installed
+on the target host. For more information about Docker Engine, see the
+[documentation][dk-doc].
+
+  [dk]: https://www.docker.com/
+  [dk-doc]: https://docs.docker.com/engine/
+
+Running this image without specifying a command will display this help message:
+
+    docker run elicocorp/odoo:10.0
+
+The easiest way to use this image is to run it along with a [PostgreSQL][pg]
+image.
+
+  [pg]: https://hub.docker.com/_/postgres/
+
+**Note:** The [PostgreSQL image][ec-pg] of Elico Corp can be used as well.
+
+  [ec-pg]: https://hub.docker.com/r/elicocorp/postgres/
+
+Below is an example of a simple `docker-compose.yml` to use this image.
+For more information about Docker Compose, see the [documentation][dc-doc].
+
+  [dc-doc]: https://docs.docker.com/compose/
+
+    version: '3.3'
+    services:
+
+      postgres:
+        image: postgres:9.5
+        environment:
+          - POSTGRES_USER=odoo
+        network_mode: bridge
+
+      odoo:
+        image: elicocorp/odoo:10.0
+        command: start
+        ports:
+         - 127.0.0.1:8069:8069
+        links:
+         - postgres:db
+        environment:
+         - ODOO_DB_USER=odoo
+        network_mode: bridge
+
+**Note 1:** With this configuration, Odoo will be accessible at the following
+URL *only* from the local host: <http://127.0.0.1:8069>
+
+It is possible to publish it following one of these options:
+
+1. map a reverse-proxy to `127.0.0.1:8069`
+2. remove `127.0.0.1:` in order to publish the port `8069`
+
+**Note 2:** With this configuration:
+
+1. Odoo is running without master password
+2. `odoo` PostgreSQL user is a superuser who doesn't require any password
+
+See [Security] section for more info.
+
+**Note 3:** With this configuration, all the data will be lost once the
+containers are stopped.
+
+See [Data persistency] section for more info.
+
+## Security
+It is recommended to:
+
+1. set a master password for Odoo using `ODOO_ADMIN_PASSWD`
+2. start PostgreSQL with a different superuser (e.g. `postgres`)
+3. give the superuser a password using `POSTGRES_PASSWORD`
+4. create a separate PostgreSQL user for Odoo (e.g. `odoo`) with his own
+password and specify it using `ODOO_DB_PASSWORD`
+
```

Comment:
> 5. Change the default port(s) of Odoo

This is only valid in a "complex" deployment scenario with reverse-proxy which we don't describe in this doc (or at least I didn't plan to, though we could simply refer to a more generic doc). Plus the port is actually only opened for localhost so it's really an optional security. I will add it in another section.

> CREATE database odoo WITH OWNER odoo;

1. This is totally optional and actually is not improving security but lowering it
2. Owner must be the superuser `postgres`, not `odoo`, so that `odoo` user cannot access this DB whose only purpose is to allow `odoo` to login with `psql`